src/main/java/io/odpf/firehose/consumer/Message.java

```diff
@@ -41,6 +42,41 @@ public Message(byte[] logKey, byte[] logMessage, String topic, int partition, lo
         this.offset = offset;
     }
 
+    /**
+     * Instantiates a new Message without providing errorType.
+     *
+     * @param logKey
+     * @param logMessage
+     * @param topic
+     * @param partition
+     * @param offset
+     * @param headers
+     * @param timestamp
+     * @param consumeTimestamp
+     */
+    public Message(byte[] logKey, byte[] logMessage, String topic, int partition, long offset, Headers headers, long timestamp, long consumeTimestamp) {
```

Comment:
Initialize errorinfo with default constructor.

Reply:
previously, we use `@AllArgsConstructor` annotation, will remove this instead

Reply:
we already have `@AllArgsConstructor` annotation for constructor with all agrs
this constructor is intended to create a message without error info,
because not every message creation need error info
but will change the implementation instead 
will call the all args constructor rather than  initialize the field manually one by one

Reply:
Ok. so from where this constructor is being called? They will not have ErrorInfo initialised.

Reply:
there is one from FirehoseConsumer, the the part that kafka ConsumerRecords converted to Messages
https://github.com/odpf/firehose/blob/6151819bd0094ac1f0ba9eb360f6148d0a136166/src/main/java/io/odpf/firehose/consumer/GenericConsumer.java#L66

Reply:
Ok. i guess it's fine, if we are relying on this to be null while checking.